app/services/topics_service.rb

```diff
@@ -5,31 +5,46 @@
 class TopicsService
   GOVUK_HOMEPAGE_CONTENT_ID = "f3bbdec2-0e62-4520-a7fd-6ffd5d36e03a"
   CACHE_OPTIONS = { expires_in: 5.minutes, race_condition_ttl: 10.seconds }.freeze
-  TOPICS_TIMEOUT = 4.seconds
+  TOPIC_INDEX_TIMEOUT = 4.seconds
 
-  def topics
+  def topic_index
```

Comment:
I'm a little confused about the name change from all_topics to index. Does it mean something different?

Reply:
So using the word `topics` or `all_topics` kind of implied an array, whereas the return value is actually a lookup table or `index` and the rest of the code uses it in this way. We thought `index` was a little easier to comprehend than `lookup_table`.

Reply:
Sure, lookup_table == 👎 - do be cautious of slipping into hungarian notation.

As an outsider index doesn't really tell me more than topics, but I know I'm only bikeshedding so don't worry about it.

Reply:
Ah, I see what you mean. Really we struggled to come up with a name for it, so probably the best thing about it is that it suggests it's not an array!